app/views/coronavirus_local_restrictions/_devolved_nation.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="covid-local-restrictions__devolved-nation-guidance">
```

Comment:
Each of these templates has a class at the top that doesn't appear anywhere in this PR - are these classes real? If not I'd suggest removing them.

Reply:
@andysellick Should we keep the `div`?

Reply:
It's probably not doing anything, because by default it has no margin or padding, so probably safe to remove? Try it first, obvs.

Reply:
We backend devs are a little bit nervous as the pages are working/look as expected at the moment. Will ask @DilwoarH when he gets back.

Reply:
Fair enough. I'm pretty confident it'll be fine - although it's absolutely fine to leave them in for now, could look at it later, in no way a blocking issue.

Reply:
Thanks! We're at the "if it ain't broke, don't fix it", part of the day!

Reply:
This can be removed - I had them in there in case we needed some specific styling. We can do this post-release.